cytoscape-supportimages.js

```diff
@@ -693,18 +710,32 @@
         var r = this;
 
         // get image, and if not loaded then ask to redraw when later loaded
-        var img = this.getCachedImage(supportImage.url, function(evt) {
+        var img = this.getCachedImage(supportImage, function(evt) {
             var resource = evt.currentTarget;
             var w = resource.width;
             var h = resource.height;
             supportImage.resourceW = w;
             supportImage.resourceH = h;
             supportImage.bounds.width = supportImage.bounds.width || w;
             supportImage.bounds.height = supportImage.bounds.height || h;
+
+            if(img.readyState >= 0){
+              img.addEventListener('canplaythrough', function(){
+                img.play();
+              });
+
+              function animateFrames(){
+                context.drawImage(img, supportImage.bounds.x, supportImage.bounds.y, supportImage.bounds.width, supportImage.bounds.height);
```

Comment:
Needs to include the part that draws the controls for resize too.
When a video is unlocked and selected the resize controls stays underneath the video frame.

Reply:
Good catch, it is fixed.  Thank you!